autometrics/README.md

```diff
@@ -115,8 +115,9 @@ fn main() {
 
 #### Metrics Libraries
 
-Configure the crate that autometrics will use to produce metrics by using one of the following feature flags:
+**Required:** Configure the crate that autometrics will use to produce metrics by using one of the following feature flags:
```

Comment:
maybe we should also document what the prioritization order is when multiple are enabled

Reply:
I changed it so it'll use all of them if multiple are enabled